components/twitch/twitch.app.js

```diff
@@ -0,0 +1,107 @@
+const axios = require("axios");
+const crypto = require("crypto");
+
+module.exports = {
+  type: "app",
+  app: "twitch",
+  propDefinitions: {
+    streamerLoginNames: {
+      type: "string[]",
+      label: "Streamer Login Names",
+      description:
+        "Enter the login names of the streamers whose streams you want to watch.",
+    },
+    game: {
+      type: "string",
+      label: "Game Title",
+      description: "Watch for live streams about the specified game.",
+    },
+    language: {
+      type: "string",
+      label: "Stream Language",
+      description:
+        'Watch for games streamed in the specified language. A language value must be either the ISO 639-1 two-letter code for a supported stream language or "other".',
+    },
+    max: {
+      type: "integer",
+      label: "Max Items",
+      description:
+        "The maximum number of items to return at one time. Streams are returned sorted by number of viewers, in descending order. Videos and Clips are returned sorted by publish date.",
+    },
```

Comment:
We can add an additional `min` parameter so that `min: 1`.

Reply:
Done.